lib/ansible/module_utils/network/ios/facts/l3_interfaces/l3_interfaces.py

```diff
@@ -50,7 +53,7 @@ def populate_facts(self, connection, ansible_facts, data=None):
         objs = []
 
         if not data:
-            data = connection.get('show running-config | section ^interface')
+            data = self.get_acl_data(connection)
```

Comment:
Can we reuse the logic in https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/network/ios/ios.py#L104-L108?